pymagicc/io.py

```diff
@@ -192,6 +190,12 @@ def postprocess_edge_cases(value):
             except TypeError:
                 metadata[metadata_key] = nml["THISFILE_SPECIFICATIONS"][k]
 
+        # One input file has annualsteps == 0, but annual timesteps
```

Comment:
which file? can we just fix that?

Reply:
`TESTSCEN7.SCEN7`. I tried updating the 0 to a 1, but the test_in_file_read_write_functionally_identical test failed because the L2483 wrote the file to disk with annualsteps = 0

Reply:
`TESTSCEN7.SCEN7` doesn't have annual timesteps though?

Reply:
Well depends on what that actually means. It does have 1 value per year for the years present in the files

Reply:
ok delete this comment then we're good